source/_integrations/frontier_silicon.markdown

```diff
@@ -17,49 +17,20 @@ This integration provides support for Internet Radios based on the [Frontier Sil
 * Medion: [Medion Radios]
 * Silvercrest: [SIRD 14 C2]
 * Teufel: [Radio 3sixty(2019)]
+* Roberts: [Roberts Stream 94i]
 * Some models from: Auna, Technisat, Revo, Pinell, Como Audio
 
-This integration was developed and tested with a Hama [DIR3110] and a Medion [MD 87466].
+This integration was developed and tested with a [Roberts Stream 94i].
 
 ## Configuration
 
-Your Frontier Silicon based device should be automatically discovered by Home Assistant. The auto-discovery service assumes that the device uses the default PIN code: *1234*. If you have changed the PIN code, the auto-discovery will fail as Home Assistant will not be able to connect to the device. You can set the PIN code of your device (depending on manufacturer) under:
+The integration supports automatic discovery of your Internet Radio. If you need to setup the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting http://[host]:[port]/device.
```

Comment:
```suggestion
The integration supports automatic discovery of your Internet Radio. If you need to set up the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting `http://[host]:[port]/device`.
```